src/sentry/static/sentry/app/components/events/interfaces/spans/index.tsx

```diff
@@ -12,22 +13,46 @@ type PropType = {
   event: SentryTransactionEvent;
 };
 
-class SpansInterface extends React.Component<PropType> {
+type State = {
+  searchQuery: string | undefined;
+};
+
+class SpansInterface extends React.Component<PropType, State> {
   static propTypes = {
     event: SentryTypes.Event.isRequired,
   };
+
+  state: State = {
+    searchQuery: undefined,
+  };
+
+  handleSpanFilter = (searchQuery: string) => {
+    this.setState({
+      searchQuery: searchQuery || undefined,
+    });
+  };
+
   render() {
     const {event} = this.props;
 
     return (
-      <Panel>
-        <PanelHeader disablePadding={false} hasButtons={false}>
-          {t('Trace View - This Transaction')}
-        </PanelHeader>
-        <PanelBody>
-          <TraceView event={event} />
-        </PanelBody>
-      </Panel>
+      <div>
+        <SearchBar
+          defaultQuery=""
+          query={this.state.searchQuery || ''}
+          placeholder={t('Filter on spans')}
+          onSearch={this.handleSpanFilter}
+        />
```

Comment:
In other parts of the application the panel header contains the search controls. For example Settings > Members

Reply:
@markstory looks like this is inline with the latest designs for the trace view on Figma. @doralchan will be killing the panel